release.config.js

```diff
@@ -0,0 +1,163 @@
+const { execSync } = require('child_process')
+
+const plugins = [
+  [
+    '@semantic-release/commit-analyzer',
+    {
+      preset: 'conventionalcommits',
+      releaseRules: [
+        {
+          type: 'add',
+          release: 'minor',
+        },
+        {
+          type: 'fix',
+          release: 'patch',
+        },
+        {
+          type: 'change',
+          release: 'minor',
+        },
+        {
+          type: 'deprecate',
+          release: 'minor',
+        },
+        {
+          type: 'remove',
+          release: 'minor',
+        },
+        {
+          type: 'security',
+          release: 'patch',
+        },
+        {
```

Comment:
Maybe we don't even need the types below unless we want to use one of them to not show up in the changelog

Reply:
I tried `release: false` but that caused an error